### PR TITLE
Use different IDs but same names for both forms so server can use sam…

### PR DESCRIPTION
**Pull request by Andrewmob2703-zz**
…e constants to get data.
Also:
Remove nickname box from both forms, and 
Use form's own submit button to submit form as normal

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -5,15 +5,8 @@
...
     <title>BIPOC Blog</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
- 
-    <style>
-      /* Form is hidden by default. */
-      .hidden {
-        display: none;
-      }
-    </style>
```
Comment (Andrewmob2703-zz) [inline portfolio/src/main/webapp/index.html:15]: took this to css file

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -65,31 +55,30 @@ <h1>Blog</h1>
...
-          <textarea id="text-input-reply" rows="6" cols="60" placeholder="What's on your mind..."></textarea>
+        <form id="reply-form" enctype="multipart/form-data">
+          <textarea id="reply-text-input" name="text-input" rows="3" cols="60" placeholder=""></textarea>
           <br>
-          <div id="hidden-reply">
-            <input type="text" id="parentID" value="0"><br>
-            <input type="text" id="tags-reply" value="#general"><br>
+          <div class="hidden">
+            <input type="text" id="reply-parentID" name="parentID" value="0"><br>
+            <input type="text" id="reply-tag" name="tags" value="#general"><br>
```
Comment (martiKallas) [inline portfolio/src/main/webapp/index.html:75]: Just a thought - should this value be assigned as the Parent comment's tag? Does that already happen somewhere?

**Review by martiKallas — APPROVED** [review]
LGTM

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (Andrewmob2703-zz) [reply portfolio/src/main/webapp/index.html:75]: Yeah it happens inside the handleReplies function in script.js

**Pull request merged**